tdp/cli/commands/plan/test_dag.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+from click.testing import CliRunner
+from .dag import dag
+from ..init import init
+
+
+def test_tdp_plan_dag(collection_path, database_dsn_path, vars):
+    args = [
+        "--collection-path",
+        collection_path,
+        "--database-dsn",
+        database_dsn_path,
+        "--vars",
+        vars,
+    ]
+    runner = CliRunner()
+    runner.invoke(init, args)
```

Comment:
Why don't you check the return code of the `init` command?

Reply:
becuse this test is testing `dag`, not `init`.  For `init` I have a dedicated test.

Reply:
Yes `init` has its own test but if the `init` is in error, it doesn't make sense to continue the test and run the next command. In the error output it will show us that the following command is in error when it is the `init` which is in question.

Reply:
This is not a unit test, you perform a `init` and a second command. If you want to do just the second command and do a real unit test you must mock the `init` command and not call the real one.

Reply:
agree, it is not a unit test. Ok, I'll change